libs/agno/agno/agent/agent.py

```diff
@@ -444,7 +444,7 @@ def __init__(
         store_tool_messages: bool = True,
         store_history_messages: bool = True,
         knowledge: Optional[Knowledge] = None,
-        knowledge_filters: Optional[Dict[str, Any]] = None,
+        knowledge_filters: Optional[Any] = None,
```

Comment:
Instead of making type as `Any` can we define a `KnowledgeFilters` type alias that explicitly lists allowed shapes (e.g. `dicts`, `FilterExpr`, lists of those)?

Reply:
Yes and we should continue to support the old way